drakrun/drakrun/draksetup.py

```diff
@@ -83,6 +83,54 @@ def check_root():
         return True
 
 
+@click.command(help='Cleanup the changes made by draksetup')
+def cleanup():
+    if not check_root():
+        return
+
+    install_info = InstallInfo.try_load()
+
+    if install_info is None:
+        logging.error("The cleanup has been performed")
+        return
+
+    backend = get_storage_backend(install_info)
+    vm_ids = get_all_vm_conf()
+
+    for vm_id in vm_ids:
+        delete_vm_conf(vm_id)
+        vm = VirtualMachine(backend, vm_id)
+        vm.destroy()
+
+        net_enable = int(conf['drakrun'].get('net_enable', '0'))
+        out_interface = conf['drakrun'].get('out_interface', '')
+        dns_server = conf['drakrun'].get('dns_server', '')
+
+        delete_vm_network(vm_id=vm_id, net_enable=net_enable, out_interface=out_interface, dns_server=dns_server)
+        if net_enable:
+            stop_dnsmasq(vm_id=vm_id)
+
+    if install_info.zfs_tank_name is not None:
+        backend.delete_zfs_tank()
+        volume_path = os.path.join("/", "dev", "zvol", install_info.zfs_tank_name, "tmp-part2")
+        try:
+            subprocess.check_output(f'umount {volume_path}', shell=True)
+            logging.info("Unmounted")
+        except subprocess.CalledProcessError as ext:
+            logging.info(ext.stdout)
+        tmp_snap = shlex.quote(os.path.join(install_info.zfs_tank_name, "tmp"))
+        try:
+            subprocess.check_output(f"zfs destroy {tmp_snap}", shell=True)
+        except subprocess.CalledProcessError:
+            pass
+
+    logging.info("Deleting postinstall files")
+    for _ in os.listdir(PROFILE_DIR):
+        safe_delete(os.path.join(PROFILE_DIR, _))
```

Comment:
We can just glob profile directory and delete everything

Reply:
isn't it happening like this only? Or you are telling to delete the profile directory and recreate it?